client/tx/factory.go

```diff
@@ -47,7 +47,7 @@ type Factory struct {
 }
 
 // NewFactoryCLI creates a new Factory.
-func NewFactoryCLI(clientCtx client.Context, flagSet *pflag.FlagSet) Factory {
+func NewFactoryCLI(clientCtx client.Context, flagSet *pflag.FlagSet) (Factory, error) {
```

Comment:
this should also updated in changelog under API breaking? cc: @julienrbrt

Reply:
Yes!

Reply:
Is there anything else I need to add? It will be helpful if you can give me a guide.

Reply:
A simple mention in the API breaking section of the CHANGELOG sufficient. E.g.
> (client) [#]() `NewFactoryCLI` now returns an error, in addition to the `Factory`.